doc/src/manual/arrays.md

```diff
@@ -872,3 +872,22 @@ julia> r
  -1.20921  -0.383393
   0.0      -0.910506
 ```
+
+### Zero-dimensional arrays
```

Comment:
At https://github.com/JuliaLang/julia/pull/27769 I've noticed we'd better move all headings up by one level. So better use `##`, that way it's already consistent if we merge my PR soon.